include/hashinator/hashinator.h

```diff
@@ -463,6 +463,28 @@ class Hashmap {
       }
    }
 
+   // Dangerous methods for exposing internals
+   template <bool warn = true>
+   HASHINATOR_HOSTDEVICE MapInfo* expose_mapinfo() noexcept {
+      if constexpr(warn) {
+         printf("Warning, exposing Hashmap internal info struct!\n");
+      }
+      return _mapInfo;
+   }
+   template <bool warn = true>
+   HASHINATOR_HOSTDEVICE hash_pair<KEY_TYPE, VAL_TYPE>* expose_bucketdata() noexcept {
+      if constexpr(warn) {
+         printf("Warning, exposing Hashmap internal bucket data!\n");
+      }
+      return buckets.data();
+   }
+   HASHINATOR_HOSTDEVICE inline KEY_TYPE expose_emptybucket() const noexcept {
```

Comment:
There is already a method for this right?
https://github.com/kstppd/hashinator/blob/426a1c8924332a1025246503565d6206182a8743/include/hashinator/hashinator.h#L449